tests/test_playbooks/usergroup.yml

```
@@ -0,0 +1,131 @@
+---
+- hosts: fixtures
```

with the merge of #531 this should be `localhost`
```suggestion
- hosts: localhost
```